src/backend/joanie/tests/core/test_api_admin_course_runs.py

```diff
@@ -120,22 +124,36 @@ def test_admin_api_course_runs_get(self):
 
         response = self.client.get(f"/api/v1.0/admin/course-runs/{course_run.id}/")
 
-        self.assertEqual(response.status_code, 200)
-        content = response.json()
-        self.assertEqual(content["id"], str(course_run.id))
-        self.assertEqual(
-            content["start"], course_run.start.isoformat().replace("+00:00", "Z")
-        )
-        self.assertEqual(
-            content["end"], course_run.end.isoformat().replace("+00:00", "Z")
-        )
-        self.assertEqual(
-            content["enrollment_start"],
-            course_run.enrollment_start.isoformat().replace("+00:00", "Z"),
-        )
+        self.assertEqual(response.status_code, HTTPStatus.OK)
+
         self.assertEqual(
-            content["enrollment_end"],
-            course_run.enrollment_end.isoformat().replace("+00:00", "Z"),
+            response.json(),
+            {
+                "id": str(course_run.id),
+                "start": format_date(course_run.start),
+                "end": format_date(course_run.end),
+                "enrollment_start": format_date(course_run.enrollment_start),
+                "enrollment_end": format_date(course_run.enrollment_end),
+                "course": {
+                    "code": course_run.course.code,
+                    "title": course_run.course.title,
+                    "id": str(course_run.course.id),
+                    "state": {
+                        "priority": course_run.course.state["priority"],
+                        "datetime": format_date(course_run.course.state["datetime"])
+                        if course_run.course.state["datetime"]
+                        else None,
```

Comment:
If I remember correctly, format_date handles None as a parameter.
```suggestion
                        "datetime": format_date(course_run.course.state.get("datetime")),
```

Reply:
Thank's I'll take a look